typings/pypandoc/__init__.pyi

```diff
@@ -0,0 +1,5 @@
+import pathlib
+
+def convert_file(
```

Comment:
Can you explain to me what's happening here? I see it being used to convert to docx, but I'm not seeing any return statement in this.